ReactAndroid/src/main/java/com/facebook/react/bridge/queue/ReactQueueConfigurationSpec.java

```diff
@@ -44,10 +48,12 @@ public static Builder builder() {
   }
 
   public static ReactQueueConfigurationSpec createDefault() {
+    MessageQueueThreadSpec spec = Build.VERSION.SDK_INT < 20 ?
```

Comment:
We were seeing the issue on KitKat (specifically API 19, don't have API 20) but not Marshmallow (API 21+). Changed this check to `< 21` so that we cover all of KitKat.